src/hooks/useSelectChain.ts

```diff
@@ -21,6 +24,10 @@ export default function useSelectChain() {
 
       try {
         await switchChain(connector, targetChain)
+        if (isSupportedChain(targetChain)) {
```

Comment:
Instead of checking `isSupportedChain`, I think that L20 could restrict usage to just `async (targetChain: SupportedChainsType) => {`. wdyt?

Reply:
I believe we'd still want the behaviour `await switchChain(connector, targetChain)` to occur on unsupported chains, just not adding them as a query parameter